libipl/libipl.H

```diff
@@ -82,6 +89,16 @@ void ipl_log(int loglevel, const char *fmt, ...);
 
 void ipl_set_error_callback_func(ipl_error_callback_func_t fn);
 
+/*
+ * @Brief Process PLAT error to add callout/deconfig and callback
```

Comment:
- Look's statement is not correct. you are handling HWP and Plat error both right?
- @Brief -> @brief?